app/src/test/shared/components/LoginFormTests.tsx

```diff
@@ -1,132 +0,0 @@
-    });
-
-    it("renders validation errors", () => {
-        const errors = {
-            email: "Blah blah"
-        };
-        const rendered = shallow(<LoginFormComponent {...mockFormProperties(form, errors)} />);
-        const validationErrors = rendered.find(ValidationError);
-        let hasMessage: boolean = false;
-        validationErrors.forEach(x => {
-            hasMessage = hasMessage || x.prop("message") == "Blah blah";
-        });
-        expect(hasMessage).to.equal(true, "Expected there to be at least one ValidationError containing the message 'Blah blah'");
-    });
-
-    it("authenticates when form is submitted", (done: DoneCallback) => {
-        const token = "TOKEN";
-        const spy = sandbox.sinon.spy(contribAuthStore, "logIn");
-
-        mockFetcher(Promise.resolve(promiseJSON({ access_token: token })));
-        form.change({
-            email: "an@email",
-            password: "not-real"
-        });
-        checkSubmit(form, done, sandbox, _ => {
-            expect(spy.called).to.be.true;
-            expect(spy.args[0][0]).to.equal(token);
-            spy.restore();
-        });
-
-    });
-
-    it("displays error when credentials are wrong", (done: DoneCallback) => {
-        mockFetcher(Promise.resolve(promiseJSON({ error: "Wrong password" })));
-        form.change({
-            email: "an@email",
-            password: "not-real"
-        });
-        checkSubmit(form, done, sandbox, spy => {
-            actionHelpers.expectOrderedActions(
-                spy,
-                [{ action: "Login_test/submitFailed", payload: "Your username or password is incorrect" }],
-                numberOfSubmissionActions
-            );
-        });
-    });
-
-    it("displays error when login fails with internal error", (done: DoneCallback) => {
-        mockFetcher(Promise.resolve(promiseJSON({
-            errors: [{
-                "code": "unexpected-error",
-                "message": "Some message"
-            }]
-        })));
-        form.change({
-            email: "an@email",
-            password: "not-real"
-        });
-        checkSubmit(form, done, sandbox, spy => {
-            actionHelpers.expectOrderedActions(
-                spy,
-                [{ action: "Login_test/submitFailed", payload: "An error occurred logging in" }],
-                numberOfSubmissionActions
-            );
-        });
-    });
-
-    it("displays error when login fails", (done: DoneCallback) => {
-        mockFetcher(Promise.reject(true));
-        form.change({
-            email: "an@email",
-            password: "not-real"
-        });
-        checkSubmit(form, done, sandbox, spy => {
-            actionHelpers.expectOrderedActions(
-                spy,
-                [{ action: "Login_test/submitFailed", payload: "An error occurred logging in" }],
-                numberOfSubmissionActions
-            );
-        });
-    });
-});
```

Comment:
Are there tests for the new log in form somewhere?

Reply:
montagu-webapps/app/src/test/shared/components/Login/LoginFormTests.tsx